h/views/api_config.py

```diff
@@ -89,3 +89,56 @@ def wrapper(wrapped):
         return wrapped
 
     return wrapper
+
+
+class AngularRouteTemplater(object):
+    """
+    Create Angular-compatible templates for named routes.
+
+    The template format here is designed to be compatible with ``ngResource``.
+    These templates are of the form:
```

Comment:
I'll note that the client doesn't actually use `ngResource` any more. However, changing the URL param format to something else would be a breaking change that doesn't really provide any value. Colon-prefixed params are also quite ubiquitous in other front-end frameworks.

Reply:
Yup: I saw that we [stopped using `ngResource` in the client](https://github.com/hypothesis/client/commit/0d9217ec4224d62fd53ba91d68b6677f9714f5de), but figured that describing the URLs as `ngResource`-compatible was the easiest way to express the format (and link to some kind of documentation). Does that sound fair, or do you think some other form of documentation would be more helpful?

Reply:
Yes, I think this is fine. I'm not aware of a more well-known reference for this URL syntax.